hadoop-hdds/framework/pom.xml

```diff
@@ -29,7 +29,16 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
   <name>Apache Ozone HDDS Server Framework</name>
   <packaging>jar</packaging>
 
+  <properties>
+    <ozone-common.version>1.3.0-SNAPSHOT</ozone-common.version>
+  </properties>
+
   <dependencies>
+    <dependency>
+      <groupId>org.apache.ozone</groupId>
+      <artifactId>ozone-common</artifactId>
+      <version>${ozone-common.version}</version>
+    </dependency>
```

Comment:
Please do not introduce backwards dependency (`hdds -> ozone`).  If we want to access some code currently in `ozone` from both, we should move it to `hdds`.

Reply:
There is a need for `org.apache.hadoop.ozone.om.OMConfigKeys`. Where exactly do you propose it should be moved to make the most sense out of this?

Reply:
`org.apache.hadoop.ozone.om` package (`om` part yet to be created but at same level where `org.apache.hadoop.hdds.scm.ScmConfigKeys` resides) from `common` module from `hadoop-hdds` (`hdds-common` artifact id) sounds reasonable enough, but there are quite a few moving parts together with `org.apache.hadoop.ozone.om.OMConfigKeys`.

Reply:
@adoroszlai @xBis7 Thanks for the suggestions! Check for `ScmConfigKeys` is enough right now - it might not be the best future proof solution but it is still better than moving tons of classes just in order to use `OMConfigKeys`.

Reply:
@adoroszlai This might be exactly what I was looking for. Thanks for suggestion, will try to expand on it!